src/database/sqlx-mysql/src/repositories/mysql_account_repository.rs

```diff
@@ -57,7 +50,7 @@ impl AccountRepository for MySqlAccountRepository {
             account_model.origin as AccountOrigin,
             account_model.registered_at,
         )
-        .execute(&mut **mysql_transaction)
+        .execute(tr.connection_mut())
```

Comment:
The goal of all this was to simplify the use here. I played with `DerefMut` hoping to simplify it to `&mut tr` but didn't get it as succinct.